pandas/_libs/tslibs/period.pyx

```diff
@@ -1236,46 +1236,64 @@ cdef list extra_fmts = [(b"%q", b"^`AB`^"),
 cdef list str_extra_fmts = ["^`AB`^", "^`CD`^", "^`EF`^",
                             "^`GH`^", "^`IJ`^", "^`KL`^"]
 
+cdef inline int idx_first_nonfiscal_fmt = 3
```

Comment:
Appears this is only used once so probably better to inline directly where used

Reply:
ok